src/Adapter/Order/AbstractOrderHandler.php

```diff
@@ -275,4 +275,80 @@ protected function clearTemporarySpecificPrices(array $temporarySpecificPrices):
             $specificPrice->delete();
         }
     }
+
+    /**
+     * Create a specific price, or update it if it already exists
+     *
+     * @param Number $priceTaxIncluded
+     * @param Number $priceTaxExcluded
+     * @param int $productQuantity
+     * @param Order $order
+     * @param Product $product
+     * @param $combination
+     *
+     * @return SpecificPrice
+     *
+     * @throws PrestaShopException
+     * @throws \PrestaShopDatabaseException
+     */
+    protected function updateSpecificPrice(
+        Number $priceTaxIncluded,
+        Number $priceTaxExcluded,
+        int $productQuantity,
+        Order $order,
+        Product $product,
+        $combination
+    ): ?SpecificPrice {
```

Comment:
I don't think you need to return the object, it's never used As long as it's correctly updated in database everything is fine